roles/ansible_service_broker/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+ansible_service_broker_image: "ansibleplaybookbundle/ansible-service-broker:latest"
```

Comment:
i think instead of image we should be using the image prefix and image version. unless we expect the image name to be different than `ansible-service-broker`. It follows the same pattern we use with all the other components